sources/packages/web/src/types/contracts/StudentContract.ts

```diff
@@ -97,36 +97,48 @@ export interface StudentDetail {
 /**
  *  Student uploader interface
  */
-export interface StudentFileUploaderForm {
+export interface StudentFileUploaderInfoAPIInDTO {
```

Comment:
Are we planning to move these DTOs inside http/dto/ in any upcoming PR or any of the APIReorganize story?

Reply:
APIReorganize story. These refactors were too far in this PR already, IMO.